.github/workflows/ci.yml

```diff
@@ -60,24 +60,19 @@ jobs:
 
```

Comment:
Let's also change all the `Install dependencies` steps to use the [immutable](https://yarnpkg.com/cli/install#immutable) flag: 

`run: yarn install --immutable`

Reply:
Additionally, the `turbo` `server-token` key should use `env.TURBO_TOKEN` instead of `secrets.TURBO_TOKEN`:

```
server-token: ${{ env.TURBO_TOKEN }}
```